### DISCUSSION TITLE
nmake error

### DISCUSSION THREAD

**Discussion by SORRY4DAWAIT** [gucci-on-fleek/lockdown-browser#56] [Q&A]
Getting this error and I don't know why, any help would be appreciated 
![ss](https://github.com/gucci-on-fleek/lockdown-browser/assets/160080562/c7773f8e-1a79-43ad-a149-dcfd3f27e254)

**Comment by gucci-on-fleek** [comment]
Did you install the [Visual Studio C++ Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools)?

From [the main page](https://github.com/gucci-on-fleek/lockdown-browser?tab=readme-ov-file#system-requirements):

> ## System Requirements
>   - Windows 10 **Pro** or **Enterprise**
>   - [Visual Studio C++ Tools](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools)
>   - [git](https://git-scm.com/download/win)
